specification/_types/query_dsl/fulltext.ts

```diff
@@ -271,24 +272,23 @@ export class QueryStringQuery extends QueryBase {
 /**
  * Query flags can be either a single flag or a combination of flags, e.g. `OR|AND|PREFIX`
  * @doc_id supported-flags
- * @codegen_names single, multiple
  */
-export type SimpleQueryStringFlags = SimpleQueryStringFlag | string
+export type SimpleQueryStringFlags = FlagsEnum<SimpleQueryStringFlag[]>
 
 export enum SimpleQueryStringFlag {
```

Comment:
Does not make a real difference as these values are currently not part of the generated schema, but I adjusted them anyways based on their actual values:


https://lucene.apache.org/core/8_9_0/queryparser/constant-values.html#org.apache.lucene.queryparser.simple.SimpleQueryParser.AND_OPERATOR

Reply:
Actually you can remove these constants altogether. They're some ancient leftovers from the very first iterations on the TypeScript spec where integer constants were added to the enums.